scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -299,26 +299,37 @@ For this example we will focus on identifying DE genes in one cell type, the `Tu
+As a reminder, the uncorrected gene expression data, in the `counts` and `logcounts` assays, correspond to data that has been merged (the first step we walked through prior to integration) but has not been corrected.
+
 ### Pseudo-bulking 
 
-Before we can compare the gene expression profiles of myoblasts in ARMS vs. ERMS patients, we will need to "pseudo-bulk" the gene counts. 
-Pseudo-bulking creates a new counts matrix that contains the total counts across all cells of a given label (e.g. cell type) for each sample. 
-This allows us to use single-cell resolution to define the labels and sum gene counts across groups of cells containing the same label, and avoids counting each cell as its own replicate. 
+Before we can compare the gene expression profiles of myoblasts in ARMS vs. ERMS samples, we will need to "pseudo-bulk" the gene counts. 
+Pseudo-bulking creates a new counts matrix that contains the total counts across all cells of a given label (e.g. cell type) for each sample ([Tung _et al.,_ 2017](https://doi.org/10.1038/srep39921)). 
+If each gene had counts from every cell, then each cell would be counted as its own replicate and we would inflate our p-values.
+By pseudo-bulking first, each gene will now have one value for each sample and we can utilize the same methods we would use if we were performing bulk RNA-seq.
 
 Pseudo-bulking is implemented prior to differential expression analysis on single-cell data for the following reasons: 
 
-- Produces larger counts, which allows us to use standard normalization and differential expression methods used by bulk RNA-sequencing. 
--  Collapses gene expression counts by sample, so that samples, rather than cells, represent replicates.
+- Produces larger counts, which allows us to use standard normalization and differential expression methods used by bulk RNA-seq. 
+- Collapses gene expression counts by sample, so that samples, rather than cells, represent replicates.
 - Masks variance within a sample to emphasize variance across samples.
 
-We are starting with the simple comparison of looking at one cell type and comparing across two groups of samples: the ERMS and ARMS subdiagnoses. 
-Our first step will be to pseudo-bulk our dataset to group our cells by cell type and by sample.
-This will create a new `SingleCellExperiment` (SCE) object that contains the pseudo-bulked counts for all genes. 
-We can subset this SCE to just contain our cell type of interest (tumor myoblasts) for input to differential expression analysis. 
+When we pseudo-bulk we want to produce a new counts matrix that has all genes and condenses all cells to include one group for each combination of cell type and sample.
+Psuedo-bulking will create a new SCE object with new dimensions that contain only the pseudo-bulked counts for all genes. 
+We can then subset this SCE to just include our cell type of interest (tumor myoblasts) for input to the DE analysis. 
```

Comment:
"with new dimensions" sort of sounds like we're adding more reduced dimensions, a thing we often do but are extremely not doing here. 
Again, not necessarily tied to this suggestion but I think it could it to be a bit more explicit in some way.

```suggestion
Psuedo-bulking will create a new SCE object that contains only the pseudo-bulked counts for all genes across all _samples_, rather than across all cells.
We can then subset this SCE to just include our cell type of interest (tumor myoblasts) for input to the DE analysis. 
```